mycroft/configuration/mycroft.conf

```diff
@@ -285,6 +285,7 @@
         "route": "/gui",
         "ssl": false
   },
+  "remote-server": "http://localhost:8000",
```

Comment:
save as above, here it seems like its serving skills, its clearer if this is moved into a gui section

Reply:
Same as above, this value is used by the skills module but I'm happy to relocate it to the gui config section